### PR TITLE
adding documentation for the filters and integration

### PR DESCRIPTION
**Pull request by AnaisUrlichs**
adding documentation for the filters and integration

### CODE REVIEW

File: docs/reference/cli/filters.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Using Integration and Filters in K8sGPT
```
Comment (AlexsJones) [inline docs/reference/cli/filters.md:1]: I would explain both filters and integrations separately first

* Filters are a way of selecting which resources you wish to be part of your default analysis
* Integrations are a way to add in additional resources to the filter list.

File: docs/reference/operator/index.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## K8sGPT Operator
```
Comment (AlexsJones) [inline docs/reference/operator/index.md:1]: I opened up a getting started PR, could either merge or reference that guide at some point

File: docs/getting-started/installation.md [hunk 3]
```diff
@@ -12,78 +12,64 @@ Ensure that you have Homebrew installed:
 - Homebrew for Linux
 Homebrew for Linux also works on WSL
 
-### Installation
+### Honebrew
```
[the remark this answers is not in the document]
Comment (AnaisUrlichs) [reply docs/getting-started/installation.md:15]: Lol, thanks

File: docs/reference/operator/index.md (hunk 2 above)
Comment (AnaisUrlichs) [reply docs/reference/operator/index.md:1]: I would want this section to detail more on the operator than "here is how you install it"

**Review by AlexsJones — APPROVED** [review]

**Pull request merged**